controllers/openstackprovisionserver_controller.go

```diff
@@ -402,7 +403,7 @@ func (r *OpenStackProvisionServerReconciler) deploymentCreateOrUpdate(
 			Containers: []corev1.Container{
 				{
 					Name:            "osp-httpd",
-					Image:           instance.Spec.ApacheImageURL,
+					Image:           r.Defaults.ApacheImageURL,
```

Comment:
Hi Oli, question about the reconcile here :
So if the contoller finds that an image is is not the right one , what will happen here, 
it will delete the old one and recreate it with the updated Image version ?

Reply:
ack